src/pulse3D/metrics.py

```diff
@@ -393,11 +393,9 @@ def __init__(
         super().__init__(rounded=rounded, **kwargs)
 
         self.baseline_widths = baseline_widths_to_use
-
-        self.velocity_start = self.baseline_widths[0]
-        self.velocity_end = self.baseline_widths[1]
-
         self.is_contraction = is_contraction
+        # always need the 10 for relaxation and 90 for contraction to compare against input baseline width
+        self.twitch_widths = set(self.baseline_widths + (10, 90))
```

Comment:
I know that baseline_widths_to_use should be a tuple based on the type hint, but I would be concerned about errors getting raised here if it is not. Can we just make this `set(self.baseline_widths) | {10, 90}`?